apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -96,10 +94,10 @@ static void ble_device_connected_cb(ble_client_ctx *ctx, ble_device_connected *d
 
 	g_conn = dev->conn_handle;
 	RMC_LOG(RMC_CLIENT_TAG, "Conn Handle : %d\n", dev->conn_handle);
-	RMC_LOG(RMC_CLIENT_TAG, "Conn Bonded : %d\n", dev->is_bonded);
-	RMC_LOG(RMC_CLIENT_TAG, "Conn CI : %d\n", dev->conn_info.conn_interval);
-	RMC_LOG(RMC_CLIENT_TAG, "Conn SL : %d\n", dev->conn_info.slave_latency);
-	RMC_LOG(RMC_CLIENT_TAG, "Conn MTU : %d\n", dev->conn_info.mtu);
+	// RMC_LOG(RMC_CLIENT_TAG, "Conn Bonded : %d\n", dev->is_bonded);
+	// RMC_LOG(RMC_CLIENT_TAG, "Conn CI : %d\n", dev->conn_info.conn_interval);
+	// RMC_LOG(RMC_CLIENT_TAG, "Conn SL : %d\n", dev->conn_info.slave_latency);
+	// RMC_LOG(RMC_CLIENT_TAG, "Conn MTU : %d\n", dev->conn_info.mtu);
```

Comment:
Please remove commented logs if they are unnecessary

Reply:
OK, I will fix it